src/serializer/types.js

```diff
@@ -37,18 +41,31 @@ export type SerializedBinding = {
   serializedValue: void | BabelNodeExpression;
   referentialized: boolean;
   modified: boolean;
+  global: boolean;
+  declarativeEnvironmentRecord?: DeclarativeEnvironmentRecord;
+  scope?: ScopeBinding
 };
 
 export type VisitedBindings = { [key: string]: VisitedBinding };
 export type VisitedBinding = {
   global: boolean;
   value: void | Value;
   modified: boolean;
+  declarativeEnvironmentRecord?: DeclarativeEnvironmentRecord;
```

Comment:
This is redundant with global, as `global === true` iff `declarativeEnvironmentRecord === undefined`. Remove `global`.